api/_mixins/HTMLHyperlinkElementUtils__HTMLAreaElement.json

```
@@ -332,7 +332,7 @@
               "version_added": "14"
             },
             "safari": {
-              "version_added": "1"
+              "version_added": "3"
```

```suggestion
              "version_added": "1"
```

Source: not looking any longer, assuming it's all the same in this file